e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -52,4 +64,26 @@ describe('Upload Files', () => {
             });
         });
     });
+
+    it('MM-T12 Loading indicator when posting images', () => {
+        const filename = 'huge-image.jpg';
+
+        // # Post an image in center channel
+        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
+        cy.get('.post-image').should('be.visible');
+        cy.postMessage('{enter}');
+
+        // # Login as otherUser
+        cy.apiLogin(otherUser);
+
+        // # OtherUser creates posts in the channel
+        cy.postMessageAs({
+            sender: otherUser,
+            message: 'post1',
+            channelId: testChannel.id,
+        });
+
+        // * Verify image is not loading for each posts
+        cy.get('.image-loading__container').should('not.be.visible');
```

Comment:
Actually, I'm not sure if this will going to be effective but I'll leave it as is.

Maybe do this several times to make sure that loading indicator is not showing up.